src/sssom/parsers.py

```diff
@@ -193,44 +194,35 @@ def parse_sssom_table(
     stream: io.StringIO = _open_input(file_path)
     sep_new = _get_seperator_symbol_from_file_path(file_path)
     df, sssom_metadata = _read_pandas_and_metadata(stream, sep_new)
-    # if mapping_predicates:
-    #     # Filter rows based on presence of predicate_id list provided.
-    #     df = df[df["predicate_id"].isin(mapping_predicates)]
-
-    # If SSSOM external metadata is provided, merge it with the internal metadata
-
-    if sssom_metadata:
-        if meta:
-            for k, v in meta.items():
-                if k in sssom_metadata:
-                    if sssom_metadata[k] != v:
-                        logging.warning(
-                            f"SSSOM internal metadata {k} ({sssom_metadata[k]}) "
-                            f"conflicts with provided ({meta[k]})."
```

Comment:
Is the warning preserved in `ChainMap`?

Reply:
No, by the nature of how ChainMap works, it picks the highest priority and disregards everything else. If requested, I can re-implement identification of discrepancies between the internal metadata and metadata passed to this function, but I think there's some comfort in knowing that ChainMap (and the accompanying docs) means that merging this data follows a clear path now that can be understood by any reader

Reply:
Yeah, I think this makes sense to me, for now;